dolphin-dnn/src/main/java/edu/snu/dolphin/dnn/layers/LayerParameter.java

```diff
@@ -24,6 +24,10 @@
 public final class LayerParameter {
   private final Matrix weightParam;
   private final Matrix biasParam;
+  private final String poolingType;
+  private final int stride;
+  private final int kernalHeight;
+  private final int kernalWidth;
```

Comment:
These parameters should not be in `LayerParameter`. `LayerParameter` is used for storing weight and bias. These parameters can be member variables of `PoolingLayer`.